index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <style>
+        html, body {height: 100%; margin: 5px; padding: 0;}
+        table {background-color: linen; font-family: "Verdana", Times, serif;}
+        th {background-color: #4CAF50;color: white;}
+        tr:nth-child(even) {background-color: #f2f2f2}
+        p {font-family: "Verdana", Times, serif;}
+    </style>
+    <link rel="stylesheet" type="text/css" href="mapStyle.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>ArcTouch Code Challenge</title>
+    <script type="text/javascript" src="prototype.js"></script>
+    <script type="text/javascript" src="utils.js"></script>
+    <script type="text/javascript" src="map.js"></script>
+    <script type="text/javascript">
+        //<![CDATA[
+
+        function getPreviousRouteList() {
+            var street = getURLParameterByName("street");
+            if (street != null) {

```

Comment:
I always used != and was not aware about the operator !== until now. It
seems to be better because it compares variable value and type.


2016-03-02 17:05 GMT-03:00 André Luiz Cardoso <notifications@github.com>:

> In index.html
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1#discussion_r54782557>
> :
>
> > +        th {background-color: #4CAF50;color: white;}
> > +        tr:nth-child(even) {background-color: #f2f2f2}
> > +        p {font-family: "Verdana", Times, serif;}
> > +    </style>
> > +    <link rel="stylesheet" type="text/css" href="mapStyle.css">
> > +	<meta name="viewport" content="width=device-width, initial-scale=1" />
> > +    <title>ArcTouch Code Challenge</title>
> > +    <script type="text/javascript" src="prototype.js"></script>
> > +    <script type="text/javascript" src="utils.js"></script>
> > +    <script type="text/javascript" src="map.js"></script>
> > +    <script type="text/javascript">
> > +        //<![CDATA[
> > +
> > +        function getPreviousRouteList() {
> > +            var street = getURLParameterByName("street");
> > +            if (street != null) {
>
> Why using != and not !== ? What's the difference?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/otaviofontana/arcTouchOtavioFontana/pull/1/files#r54782557>
> .
>